library/src/otx/backend/native/models/classification/backbones/efficientnet.py

```diff
@@ -672,6 +672,6 @@ def __new__(
 
         if pretrained:
             cache_dir = Path.home() / ".cache" / "torch" / "hub" / "checkpoints"
-            download_model(net=model, model_name=f"{model_name}", local_model_store_dir_path=str(cache_dir))
+            download_model(net=model, model_name=f"{model_name}c", local_model_store_dir_path=str(cache_dir))
```

Comment:
Thanks Ilya for the contribution. I thinks here we can not just replace this weights without a set of comprehensive experiments. 100% positive correlation with IN-top1 is not given, there could be sudden outlier which we'd like to avoid. 

To tackle that, I'd propose moving this version parameter to `init` of `EfficientNetBackbone` and forwarding it from the endpoint classes (`EfficientNetMulticlassCls` etc). Once done, the model version is configurable from model recipe yaml file. Corner case of b0 and b1 should be handled.